src/pykeen/nn/combination.py

```diff
@@ -152,6 +152,10 @@ def __init__(
     def forward(self, xs: Sequence[torch.FloatTensor]) -> torch.FloatTensor:  # noqa: D102
         return self.aggregation(super().forward(xs=xs), dim=self.dim)
 
+    # docstr-coverage: inherited
+    def extra_repr(self) -> str:  # noqa: D102
```

Comment:
Does this need to get cleaned up with the new repr stuff

Reply:
https://github.com/pykeen/pykeen/pull/988/commits/f9e3447ef55e05476c5c371ae2a48b923b127476